### ISSUE TITLE
[Export]export sample dashboard from homescreen is loading forever + downloaded format is incorrect double checked on production, working properly. not sure this is related to this PR, please confirm. also the file i downloaded is in JSON format, should be .zip instead

### ISSUE THREAD

**Issue by junlincc** [apache/superset#16730]
1. export sample dashboard from homescreen is loading forever. double checked on production, working properly. not sure this is related to this PR, please confirm. also the file i downloaded is in JSON format, should be .zip instead

https://user-images.githubusercontent.com/67837651/133664311-dbd654d3-7bb7-4d5c-b1fb-e28058193a5a.mov

_Originally posted by @junlincc in https://github.com/apache/superset/issues/16701#issuecomment-921136500_

**Comment by junlincc** [comment]
Hi @eschutho @betodealmeida could you take a look? we found it in one of our open PR and confirmed this is happening in master as well. production is working as expected. thanks!

**Comment by junlincc** [comment]
@eschutho yes, it's happening on master, without any FF configured. thank you!

**Comment by betodealmeida** [comment]
If no feature flags are configured then it will download as JSON, the ZIP is only used if `VERSIONED_EXPORT` is changed to true.
